test/scripts/e2e-kind.sh

```diff
@@ -26,8 +26,8 @@ should set TCP CLOSE_WAIT timeout
 # TO BE IMPLEMENTED: https://github.com/ovn-org/ovn-kubernetes/issues/1142
 \[Feature:IPv6DualStackAlphaFeature\]
 
-# TO BE IMPLEMENTED: https://github.com/ovn-org/ovn-kubernetes/issues/819
-Services.+session affinity
+-# TO BE IMPLEMENTED: https://github.com/ovn-org/ovn-kubernetes/issues/819
+-Services.+session affinity timeout
```

Comment:
Presumably the leading "`-`" there is a mistake? Also, if this PR is going to close #819 then this needs to link to the new issue instead.

Reply:
you are right, it can not close 819, will change it